### ISSUE TITLE
Clarify we can enable slashed zero by `"zero"` instead of `"ss19"` in README

### ISSUE THREAD

**Issue by tats-u** [microsoft/cascadia-code#764]
# Description of the new feature/enhancement (with images if possible)

There is the unified OpenType feature to enable the slashed zero: [`"zero"`](https://learn.microsoft.com/en-us/dotnet/desktop/wpf/advanced/opentype-font-features?view=netframeworkdesktop-4.8#slashed-zero).

It is adopted by JetBrains Mono and Inter. We should avoid the unstandardized way in the font installed in the standard programming font in a OS).

If we use it, we can enable it by [`font-variant-numeric: slashed-zero`](https://developer.mozilla.org/en-US/docs/Web/CSS/font-variant-numeric) without `font-feature-settings` in CSS.

# Proposed technical implementation details (optional)

**Comment by aaronbell** [comment]
Cascadia supports both `zero` and `ss19`, in cases where `zero` doesn't work.

**Comment by tats-u** [comment]
@aaronbell
Ah, you're right.

![image](https://github.com/user-attachments/assets/793ff1c8-18d2-4b11-9135-4aa0ded9cd1b)

https://codepen.io/tats-u/pen/bGPxwNL

Then, could you add "zero" at the top or bottom of "ss19"?

![](https://github.com/microsoft/cascadia-code/raw/main/images/stylistic_set.png)

**Comment by tats-u** [comment]
@aaronbell 
Now I changed the title, could you reopen this issue?

**Comment by aaronbell** [comment]
Yes. I will be doing an overhaul of the OT documentation in time.

**Comment by tats-u** [comment]
Thank you for your super quick response.
